types/src/account_config/resources/sequence_info.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Copy, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub enum AccountSequenceInfo {
+    Sequential(u64),
```

Comment:
we probably should remove this as followup, there's no such a resource in move side anymore